.github/workflows/dependabot.yml

```diff
@@ -29,7 +29,6 @@
           PR_NUMBER: ${{ github.event.pull_request.number }}
           PR_URL: ${{ github.event.pull_request.html_url }}
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
-          JIRA_ASSIGNEE: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
```

Comment:
for mongocli, i'm removing the assignee since it was being set to apix escalation

Reply:
this means that the new ticket won't have an assignee. Just flagging this to make sure we are all okay with this.

Reply:
yep, ok with that so far until the team has their own escalation, i'll make sure to ask them if they have one